api/accounts/views.py

```diff
@@ -75,3 +77,26 @@ class UserList(generics.ListAPIView):
     
 # add view to update a profile, it should only allow a profile to be updated when they have a token saying theyre logged in
 
+def sign_s3(request):
+  S3_BUCKET = os.environ.get('S3_BUCKET_NAME')
```

Comment:
This should go into the `settings.py` file and import it from there. Also, this will need to work locally

Reply:
Done! separate cotripperdev bucket set up with separate development user that has restricted access only to the cotripperdev bucket. Will send out .env file, once pull request is merged.